lib/presentation/contact_form/contact_form_screen.dart

```diff
@@ -42,134 +60,126 @@ class ContactFormState extends State<ContactFormPage> {
-              padding:
-                  const EdgeInsets.symmetric(vertical: 100.0, horizontal: 23.0),
-              child: Column(
-                mainAxisAlignment: MainAxisAlignment.center,
-                children: [
-                  const Text(
-                    "We want to know what you think!",
-                    style:
-                        TextStyle(fontWeight: FontWeight.w700, fontSize: 32.0),
-                    maxLines: 2,
-                    textAlign: TextAlign.center,
-                  ),
-                  const SizedBox(height: 35.0),
-                  TextFormField(
-                    key: _emailKey,
-                    onChanged: (value) => _emailKey.currentState!.validate(),
-                    enabled: _isEnabled,
-                    validator: (value) => _validateEmail(value),
-                    style: const TextStyle(fontSize: 20.0),
-                    decoration: const InputDecoration(
-                        suffixIcon: Icon(Icons.alternate_email),
-                        labelText: 'Email',
-                        hintText: 'johndoe@gmail.com'),
-                  ),
-                  const SizedBox(height: 25.0),
-                  TextFormField(
-                    enabled: _isEnabled,
-                    keyboardType: TextInputType.multiline,
-                    maxLines: null,
-                    minLines: 5,
-                    decoration: const InputDecoration(
-                      suffixIcon: Icon(Icons.feedback_outlined),
-                      labelText: 'Give us your feedback or request',
-                      hintText:
-                          'Give your feedback or request for starting a \ncrowdaction',
-                    ),
-                    validator: (value) {
-                      if (value == null || value.isEmpty) {
-                        return 'Please enter your feedback';
-                      }
-                      _formData['message'] = value;
-                      return null;
-                    },
-                  ),
-                  const SizedBox(height: 35.0),
-                  Row(
-                    mainAxisAlignment: MainAxisAlignment.center,
-                    children: [
-                      Expanded(
-                        child: RectangleButton(
-                          text: 'Let us know',
-                          enabled: _isEnabled,
-                          onTap: _isEnabled ? () => _validateForm() : null,
-                        ),
+    return WillPopScope(
+      onWillPop: () async {
+        ScaffoldMessenger.of(context).removeCurrentSnackBar();
+        return true;
+      },
+      child: Scaffold(
+        extendBodyBehindAppBar: true,
+        appBar: ScrollableAppBar(
+          title: 'Contact form',
+          pageScrollController: _pageScrollController,
+        ),
+        body: ScrollConfiguration(
+          behavior: const ScrollBehavior(), // TODO: use NoRippleBehavior(),

```

Comment:
Can you change this out with the NoRippleBehavior and remove the todo? I think I added the Todo before I added the no ripple behavior class.